src/api/json/catalog.json

```diff
@@ -3680,7 +3680,12 @@
     {
       "name": "Taskfile YAML Schema",
       "description": "JSON Schema for Taskfile files.",
-      "fileMatch": ["Taskfile.yaml", "Taskfile.yml"],
+      "fileMatch": [
+        "Taskfile.yaml",
+        "Taskfile.dist.yaml",
+        "Taskfile.yml",
+        "Taskfile.dist.yml"
+      ],
       "url": "https://json.schemastore.org/taskfile.json"
```

Comment:
change URL to 
"url": "https://taskfile.dev/schema.json"

Reply:
Pushed a change. Thanks